src/lib/headers.js

```diff
@@ -31,7 +31,7 @@
 const Mustache = require('mustache')
 const Enums = require('@mojaloop/central-services-shared').Enum
 const Config = require('../lib/config')
-const uriRegex = /\/(participants|parties|quotes|transfers|fxTransfers)(?:$|\/.*$)/
+const uriRegex = /(?:^.*)(\/(participants|parties|quotes|transfers|fxTransfers)(\/.*)*)$/
```

Comment:
do we need participants, parties and quotes here? isn't transfers and fxTransfers not sufficient?

Reply:
We shouldn't :thinking: not sure why this was the original regex. Removed.